tests/ov/conftest.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python2
```

Comment:
Maybe merge this file with the other conftest.py?

There is some duplicate code here: 
- RANDOM_FRAME is also defined in test_hdlc.py
- I'm not sure where the "runslow" marker is used?

Reply:
Oh I missed the fact that there was already one. And I think its stuff I ended up removing.